packages/arb-token-bridge-ui/src/components/common/PortalProject.tsx

```diff
@@ -0,0 +1,73 @@
+import Image from 'next/image'
+import { ExternalLink } from './ExternalLink'
+
+export type PortalProject = {
+  chains: string[]
+  description: string
+  id: string
+  images: { logoUrl: string; bannerUrl: string }
+  subcategories: { id: string; title: string }[]
+  title: string
+  url: string
+}
+
+export const Project = ({
+  project,
+  onClick
+}: {
+  project: PortalProject
+  onClick?: () => void
+}) => {
+  return (
+    <div className="h-full min-h-[150px] w-full overflow-hidden rounded-md border border-white/30 bg-dark hover:bg-dark-hover">
```

Comment:
looks like this `div` can be removed and its styles can be moved to `ExternalLink`?